ledger-service/http-json/src/main/scala/com/digitalasset/http/ContractsService.scala

```diff
@@ -484,14 +530,24 @@ class ContractsService(
     )
 
   private[http] def resolveTemplateIds[Tid <: domain.TemplateId.OptionalPkg](
+      jwt: Jwt,
+      ledgerId: LedgerApiDomain.LedgerId,
+  )(
       xs: OneAnd[Set, Tid]
-  ): (Set[domain.TemplateId.RequiredPkg], Set[Tid]) = {
+  )(implicit
+      lc: LoggingContextOf[InstanceUUID with RequestID]
+  ): Future[(Set[domain.TemplateId.RequiredPkg], Set[Tid])] = {
+    import scalaz.syntax.traverse._
     import scalaz.std.iterable._
-    import scalaz.syntax.foldable._
+    import scalaz.Scalaz.listInstance
+    import scalaz.Scalaz.futureInstance
```

Comment:
Don't refer to Scalaz (or scala-library, for that matter) implicits by name, use wildcards instead.

```suggestion
    import scalaz.std.list._, scalaz.std.scalaFuture._
```

Reply:
ok, thanks :)!